index.html

```diff
@@ -988,6 +989,17 @@ <h3>Concrete Lifecycle Example</h3>
   }
 }
         </pre>
+
+        <p class="note">
+Implementers that are interested in understanding more about the
+<code>proof</code> mechanism used above can learn more in
+Section <a href="#proofs-signatures"></a> and by reading the
+following specifications: Linked Data Proofs [[LD-PROOFS]], Linked Data
+Signatures [[LD-SIGNATURES]], 2018 RSA Signature Suite [[LDS-RSA2018]], and
+JSON Web Signature (JWS) Unencoded Payload Option [[RFC7797]]. A current list
+of available proof mechanisms is available in the Verifiable Credentials
```

Comment:
recommend changing to:
"A list of proof mechanisms is available in the Verifiable Credentials . . ."

Reply:
Fixed in 4816ada7ed318fa2f13888569581376a0f71804d.